.gitignore

```diff
@@ -21,6 +21,11 @@ regtests/derby.log
 regtests/metastore_db
 regtests/output/
 
+# Python stuff
+poetry.lock
+polaris-venv/
+pyproject.toml
```

Comment:
Hm - looks like `./regtests/client/python/pyproject.toml` and `./regtests/client/python/poetry.lock` are in versioned Git though.

Reply:
but they are not covered by this exclusion :thinking: Do you mean we should exclude those files or commit `poetry.lock`?

Reply:
They would be. It's just that those files are already managed by Git.
Maybe you meant
```
/poetry.lock
/pyproject.toml
```
?

Reply:
```
/p/polaris (git-ignore|✚1) $ git st
On branch git-ignore
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   regtests/client/python/poetry.lock
```

... but it's a good point in general. I've added leading slashes to avoid ambiguity.